packages/profile/src/typography/index.module.css

```diff
@@ -206,3 +205,63 @@ textarea {
   font-family: "Graphik Web", Arial, sans-serif;
   font-size: 1rem;
 }
+
+.markdown h1 {
+  line-height: 1.2;
+  font-family: "Graphik Web", Arial, sans-serif;
+  font-weight: normal;
+}
+.markdown h1.fancy {
+  font-family: "Recoleta", serif;
+}
+.markdown h1 {
+  font-size: 2.25rem;
+}
+.markdown * + h1 {
+  margin-top: 2.25rem;
+}
+
+.markdown h2 {
+  line-height: 1.3;
+  font-size: 1.5rem;
+  font-weight: 500;
+  margin-bottom: 0.5rem;
+}
+.markdown h2.fancy {
+  font-family: "Recoleta", serif;
+}
+.markdown * + h2 {
+  margin-top: 3rem;
+}
+
+.markdown h3 {
+  line-height: 1.3;
+  font-size: 1rem;
+  font-weight: 500;
+  margin-bottom: 0.5rem;
+}
+.markdown * + h3 {
+  margin-top: 2rem;
+}
+.markdown code {
+  padding: 0.2rem;
+  border-radius: 0.2rem;
+  background-color: var(--color-secondary3);
+}
+.markdown ul {
+  line-height: 1.5rem;
+}
+.markdown a {
+  text-decoration: none;
+  color: var(--color-standard__black);
+  border-bottom: 2px solid var(--color-secondary2);
+}
+.markdown a:visited {
+  color: var(--color-standard__black);
+}
+.markdown a:hover {
+  border-bottom: 2px solid var(--color-secondary3);
+}
+.markdown strong {
+  font-weight: 500;
+}
```

Comment:
`ul` and `strong` can do without the .markdown selector.
perhaps add `ol` well, making it `ul, ol`

Reply:
FIxored